packages/leva/src/components/Select/Select.tsx

```diff
@@ -10,13 +10,13 @@ function Select({
   id,
   settings,
 }: Pick<SelectProps, 'displayValue' | 'onUpdate' | 'id' | 'settings'>) {
-  const { keys, values } = settings!
+  const { keys } = settings!
 
   return (
     <SelectContainer>
-      <StyledSelect id={id} value={displayValue} onChange={(e) => onUpdate(values[+e.currentTarget.value])}>
```

Comment:
Omph I can't believe I did the `+` int conversion in the first place I hate that 😅